plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/format/SQLFormatterConfiguration.java

```diff
@@ -110,6 +112,12 @@ public boolean isFunction(String name) {
         return syntaxManager.getDialect().getFunctions().contains(name.toUpperCase(Locale.ENGLISH));
     }
 
+    public boolean isIdentifier(String name) {
+        SQLDialect dialect = syntaxManager.getDialect();
+        boolean isQuoted = dialect.isQuotedIdentifier(name);
+        return name.chars().mapToObj(c -> (char) c).allMatch(c -> dialect.validIdentifierPart(c, isQuoted));
```

Comment:
Please don't box, that pipeline step is completely redundant

```suggestion
        return name.chars().allMatch(c -> dialect.validIdentifierPart((char) c, isQuoted));
```